docs/guides/monitoring-mode.mdx

```diff
@@ -9,50 +9,63 @@ import { RunPkgCommand } from "@site/src/partials";
 
 ### Workflow
 
 1. **Feature Development**
 
-- A developer commits code to a feature or bugfix branch.
-- Optionally, a pull request (PR) is opened for review.
+   - A developer creates a feature (or bugfix) branch and commits changes.
+   - Optionally, a pull request (PR) is opened.
 
-2. **Visual Tests on the Feature Branch**
+2. **Visual Tests in CI**
 
-- Argos integrates with your test framework [via SDKs](/getting-started).
-- During CI, your test framework runs visual tests.
-- Screenshots are captured and uploaded to Argos.
+   - When the pull request is created, the CI pipeline runs automated tests.
+   - During these tests, your test framework (Playwright, Cypress, etc.) captures screenshots of the app. Argos offers [SDKs](/getting-started) for easy integration with popular frameworks.
+   - At the end of the tests, screenshots are uploaded to Argos automatically with the SDK, or manually using the Argos CLI.
 
-3. **Baseline Comparison**
+3. **Comparison with Baseline**
 
-- Argos determines the [baseline build](/baseline-build) from Git history and other criteria.
-- New screenshots are compared against the baseline.
+   - Argos receives the build containing screenshots and metadata.
+   - It automatically determines the [baseline build](/baseline-build) using Git history analysis and other criteria.
+   - The new screenshots are compared against the baseline.
 
 4. **Results & Notifications**
 
-- When the build completes:
-  - ✅ No differences → commit status set to _success_.
-  - ❌ Differences found → commit status set to _failed_, with diffs visible in the Argos dashboard.
-- On GitHub, Argos also posts a summary comment with a link to the dashboard.
+   - The build is complete once all screenshots have been compared:
+     - ✅ No differences → commit status set to _success_.
+     - ❌ Differences found → commit status set to _failed_. The changes are visible in the Argos app for review and approval.
+   - Argos notifies GitHub about the build status and posts a summary comment on the pull request.
+   - If branch protection requires passing visual tests, the PR cannot be merged until approvals are granted.
 
 5. **Approval Process**
 
-- The team reviews differences in the Argos dashboard.
-- Changes are either approved or rejected.
-- The commit status updates accordingly.
-- If branch protection requires passing visual tests, the PR cannot be merged until approvals are granted.
+   - The team reviews the changes in the Argos app:
+     - Each screenshot change can be approved or rejected.
+     - The build as a whole must then be approved or rejected and the check status on GitHub is updated accordingly.
+
+6. **Update Baseline**
+   - Once the feature branch is merged into the default branch:
+     - The CI pipeline must run on the default branch and upload screenshots to Argos.
+     - Argos receives the new build and automatically approves it.
+     - This build becomes a new baseline for future comparisons.
```

Comment:
Remove all of that that's only partially true. We have a dedicated page to explain how baseline builds are determined.